srcpkgs/font-libertine-otf/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'font-libertine-otf'
+pkgname=font-libertine-otf
+version=5.3.0
+revision=1
+_date=2012_07_02
+archs=noarch
+create_wrksrc=yes
+depends="font-util xbps-triggers"
+short_desc="Libertine Fonts - Libre multilingual font family (OTF variant)"
+maintainer="Pascal Vanier <pascal.vanier@gmail.com>"
+license="OFL-1.1"
+homepage="http://libertine-fonts.org/"
+distfiles="${SOURCEFORGE_SITE}/project/linuxlibertine/linuxlibertine/${version}/LinLibertineOTF_${version}_${_date}.tgz ${SOURCEFORGE_SITE}/project/linuxlibertine/linuxlibertine/${version}/LinLibertineTTF_${version}_${_date}.tgz"
+checksum="3eb84d13e59d390a3f0b15f93bbb590de693dd5f29c4e8083317dbbadf6fb985 24a593a949808d976850131a953c0c0d7a72299531dfbb348191964cc038d75d"
```

Comment:
put the second value into a second line with a whitespace prefix, do the same for distfiles.